hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -2321,6 +2321,16 @@ The switch to turn S3A auditing on or off.
   </description>
 </property>
 
+<property>
+  <name>ipc.server.handler.queue.size</name>
+  <value>100</value>
+  <description>
+    Indicates how many calls per handler are allowed in the queue. This value can
+    determine the maximum call queue size multiplied by multiplying the number of
```

Comment:
Can we replace `multiplied by multiplying` with `by multiplying`?

Reply:
OK. Fixed it.